src/NUnitFramework/framework/Internal/RuntimeFramework.cs

```diff
@@ -154,7 +234,7 @@ private void InitFromFrameworkVersion(Version version)
                                 switch (version.Minor)
```

Comment:
You didn't make the change and I don't want to do the change within this PR, but next time we are in this code we should clean up these nested switch statements. I can see we want to support future versions of the CF framework, but double nested If's with one case each?